File: finah-desktop-java/net/azurewebsites/finahweb/Antwoord.java
package net.azurewebsites.finahweb;

class Antwoord{
	int AntwoordExtra;
	int AntwoordInt;
	int Id;
	boolean Verzorger;
	int Vraag_Id;
	int Rapport_Id;

	public String toString(){
		return Id + ": " + Vraag_Id + "\t|" + AntwoordInt + "|" + AntwoordExtra + "\t(" + Verzorger + ")";
	}
}

File: finah-desktop-CSharp/finah-desktop-CSharp/Antwoord.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace finah_desktop_CSharp
{
    public class Antwoord
    {
        public int Id { get; set; }

        public int Vraag_Id { get; set; }

        public int Rapport_Id { get; set; }

        public int AntwoordInt { get; set; }

        public int AntwoordExtra { get; set; }

        public Boolean Verzorger { get; set; }
    }

}
